TRANSLATIONS.md

```diff
@@ -27,17 +27,24 @@ GNU Gettext utilities below.
 
 ## Preparation
 
-We use two helpers for the translations:
+You will need the [Gettext] utilities (`msginit`, `msgmerge`). Under Debian and
+Ubuntu, you can install with:
+
+```shell
+$ sudo apt install gettext
+```
+
+We use `mdbook` and two helpers for the translations:
 
 - `mdbook-xgettext`: This program extracts the English text. It is an mdbook
   renderer.
 - `mdbook-gettext`: This program translates the book into a target language. It
   is an mdbook preprocessor.
 
-Install both helpers with:
+Install everything with:
 
 ```shell
-$ cargo install mdbook-i18n-helpers
+$ cargo install mdbook mdbook-i18n-helpers
```

Comment:
I think we should leave this as it is and instead refer back to the README for how to build the book (as @henrif75 suggested). So the instructions here could start with something like:

> First, ensure you can build the book and that `mdbook serve` works (follow the instructions in the README).

Then everything will be nicely split.

Reply:
Ok, let me know if that works!